subsys/bluetooth/host/gatt.c

```diff
@@ -180,13 +183,125 @@ BT_GATT_SERVICE_DEFINE(_2_gap_svc,
 #endif
 );
 
-#if defined(CONFIG_BT_GATT_DYNAMIC_DB)
-static void sc_ccc_cfg_changed(const struct bt_gatt_attr *attr,
-			       u16_t value)
+struct sc_data {
+	u16_t start;
+	u16_t end;
+} __packed;
+
+struct gatt_sc_cfg {
+	u8_t		id;
+	bt_addr_le_t	peer;
+	struct sc_data	data;
+};
+
+#define SC_CFG_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
+static struct gatt_sc_cfg sc_cfg[SC_CFG_MAX];
```

Comment:
Should we maybe just use BT_GATT_CCC_MAX here instead?

Reply:
In the future we might have CCC loaded on demand when a device connects and BT_GATT_CCC_MAX could then be modified to only be equal to BT_MAX_CONN, so I think it is preferable to leave it as a separate define for now.